jet-proto/Cargo.toml

```diff
@@ -12,3 +12,22 @@ httparse = "1.3.4"
 http = "0.1.18"
 
 hex-literal = "0.2"
+
+[dev-dependencies]
```

Comment:
hum... where are there so many *dev-dependencies* all of sudden? dev dependencies are only required for building, but not for the actual crate itself: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

this looks like a mistake?

Reply:
I added all of these, because the tests were failing as they were not able to resolve the functions that belong to those dependencies. This all belongs to the tests folder in the jet-proto directory.